console/console-init/ui/src/components/FinishedStep/FinishedStep.tsx

```diff
@@ -34,19 +35,19 @@ const FinishedStep: React.FunctionComponent<IFinishedStepProps> = ({
 }) => {
   const [percent, setPercent] = useState<number>(0);
   const [isCompleted, setIsCompleted] = useState<boolean>(false);
-  const tick = () => {
-    if (percent < 100) {
-      setPercent(percent + 20);
-    } else {
-      if (!isCompleted) {
-        setIsCompleted(true);
-      }
-    }
-  };
+
   useEffect(() => {
-    const interval = setInterval(() => tick(), 500);
+    const interval = setInterval(() => {
+      if (percent < 100) {
+        setPercent(percent + 20);
+      } else {
+        if (!isCompleted) {
+          setIsCompleted(true);
+        }
+      }
+    }, 500);
     return () => clearInterval(interval);
-  }, [percent]);
+  }, [percent, isCompleted]);
 
   const projectDetailUrl = () => {
     if (routeDetail && projectType === "IoT") {
```

Comment:
use projectType enum for "Iot" which yiu have created.
projectType === "IoT" ==> projectType===ProjectType.IOT_PROJECT